runtime/tools/spec-builder/index.js

```diff
@@ -5,24 +5,30 @@ const curPath = process.cwd();
 
 let argv = require('yargs')
     .usage('Usage: $0 <command> [options]')
-    .command('build-spec', 'Converts custom spec json file to raw')
-    .example('$0 build-spec -f customSpec.json -o customSpecRaw.json', 'convert given file to raw and write to output')
+    .example('$0 -f customSpec.json -o customSpecRaw.json', 'convert given file to raw and write to output')
     .alias({'f' : 'file', 'o' : 'output'})
     .nargs({'f' : 1, 'o': 1})
     .describe('f', 'Load a file')
     .demandOption(['f'])
     .help('h')
     .alias('h', 'help')
-    .epilog('copyright 2020')
+    .epilog('copyright 2020 Limehchain LTD.')
     .argv;
 
-let customSpec = require(`${curPath}/${argv.file}`);
-const rawGenesis = GenesisBuilder.toRaw(customSpec);
 
-customSpec.genesis = rawGenesis;
+if(fs.existsSync(`${curPath}/${argv.file}`)){
+    let customSpec = require(`${curPath}/${argv.file}`);
 
-// set default file to write, if output file isn't provided
-const outputFile = argv.output ? argv.output : `${curPath}/customSpecRaw.json`;
-fs.writeFileSync(outputFile, JSON.stringify(customSpec, null, 2));
+    const rawGenesis = GenesisBuilder.toRaw(customSpec);
 
-console.log("succesfully converted to raw json");
\ No newline at end of file
+    customSpec.genesis = rawGenesis;
+
+    // set default file to write, if output file isn't provided
+    const outputFile = argv.output ? argv.output : `${curPath}/customSpecRaw.json`;
+    fs.writeFileSync(outputFile, JSON.stringify(customSpec, null, 2));
+
+    console.log("succesfully converted to raw json");
+}
+else{
+    console.log(`${argv.file} doesn't exist in ${curPath}`);
```

Comment:
if f.e I am passing path as file argument that looks like this:
```
spec-builder --file ./some-dir/other-dir/chain-spec.json
```
you will log only that the file does not exist in the current path of the program, but actually what it should log is that that the file does not exist in the whole path that we are checking for `${curPath}/${argv.file}`.